comps/guardrails/toxicity_detection/README.md

```diff
@@ -4,11 +4,13 @@
 
 ## Introduction
 
-Intel also provides toxicity detection model, which is lightweight, runs efficiently on a CPU, and performs well on toxic_chat and jigsaws datasets. More datasets are being fine-tuned. If you're interested, please contact abolfazl.shahbazi@intel.com.
+Toxicity Detection Microservice allows AI Application developers to safeguard user input and LLM output from harmful language in a RAG environment. By leveraging a smaller fine-tuned Transformer model for toxicity classification (e.g. DistilledBERT, RoBERTa, etc.), we maintain a both performant and lightweight guardrails microservice that is readily deployable on both Intel Gaudi and Xeon.
 
-## Training Customerizable Toxicity Model on Gaudi2
+Toxicity is defined as rude, disrespectful, or unreasonable language likely to make someone leave a conversation. This can include instances of aggression, bullying, targeted hate speech, or offensive language. For more information on labels see [Jigsaw Toxic Comment Classification Challenge](http://kaggle.com/c/jigsaw-toxic-comment-classification-challenge).
 
-Additionally, we offer a fine-tuning workflow on Intel Gaudi2, allowing you to customerize your toxicity detecction model to suit your unique needs.
+## Future Development
+
+- Add a RoBERTa (125M params) toxicity model fine-tuned on Guaidi with ToxicChat and Jigsaw dataset in an optimized serving framework.
```

Comment:
```suggestion
- Add a RoBERTa (125M params) toxicity model fine-tuned on Gaudi with ToxicChat and Jigsaw dataset in an optimized serving framework.
```

Also, do we want to add links to the datasets and model here?

Reply:
Fixed